Google.PowerShell/Common/ReportCmdletResults.cs

```diff
@@ -174,32 +180,33 @@ public GoogleAnalyticsCmdletReporter(string clientID, AnalyticsEventCategory ana
             }
         }
 
-        public void ReportSuccess(string cmdletName, string parameterSet)
+        public void ReportSuccess(string cmdletName, string parameterSet, string projectNumber)
         {
-            Report(cmdletName, parameterSet, null);
+            Report(cmdletName, parameterSet, projectNumber, null);
         }
 
-        public void ReportFailure(string cmdletName, string parameterSet, int errorCode)
+        public void ReportFailure(string cmdletName, string parameterSet, string projectNumber, int errorCode)
         {
-            Report(cmdletName, parameterSet, errorCode);
+            Report(cmdletName, parameterSet, projectNumber, errorCode);
         }
 
-        private void Report(string cmdletName, string parameterSet, int? errorCode)
+        private void Report(string cmdletName, string parameterSet, string projectNumber, int? errorCode)
         {
             // The event type will be CmdletInvocation or ProviderInvocation. The specific cmdlet and parameter set
             // used are encoded in the event's metadata.
             AnalyticsEvent eventData = new AnalyticsEvent(
                 _category.ToString(),
                 "cmdletName", cmdletName,
                 "parameterSet", (parameterSet ?? "null"),
+                "projectNumber", (projectNumber ?? "null"),
                 "errorCode", (errorCode.HasValue ? errorCode.ToString() : "null"));
 
             s_reporter.Value.ReportEvent(
                 source: "virtual.powershell",
                 eventType: "powershell",
                 eventName: eventData.Name,
                 userLoggedIn: true,
-                projectNumber: null,
+                projectNumber: projectNumber,
```

Comment:
We actually hash the project name before reporting it.  I can update the parameter to projectName instead of projectNumber to accurately reflect that. Will that be sufficient?